lib/parity/environment.rb

```diff
@@ -132,7 +132,7 @@ def run_migration?
     end
 
     def rails_app?
-      Kernel.system("command -v rake && rake -n db:migrate")
+      Kernel.system('[ -d "db/migrate" ] && [ -f "Rakefile" ]')
```

Comment:
Question. Is the existence of a `Rakefile` and a `db/migrate` directory good enough evidence that this is a Rails app?